regression/scripts/release_toss2.sh

```diff
@@ -81,15 +80,29 @@ establish_permissions
 export source_prefix="/usr/projects/$package/$pdir"
 (cd /usr/projects/$package; rm latest; ln -s $pdir latest)
 scratchdir=`selectscratchdir`
-ppn=`showstats -n | tail -n 1 | awk '{print $3}'`
+
+# ppn=`showstats -n | tail -n 1 | awk '{print $3}'`
 #build_pe=`npes_build`
 #test_pe=`npes_test`
 
-avail_queues=`mdiag -u $LOGNAME | grep ALIST | sed -e 's/.*ALIST=//' | sed -e 's/,/ /g'`
+# SLURM
+avail_queues=`sacctmgr -np list assoc user=$LOGNAME | sed -e 's/.*|\(.*dev.*\|.*access.*\)|.*/\1/' | sed -e 's/|.*//'`
+
 case $avail_queues in
   *access*) access_queue="-A access" ;;
```

Comment:
This should be 
```
access_queue="-A access --qos=access"
```

The incorrect version doesn't impact the release - it just means that the tests took longer to start.